geotrek/outdoor/locale/en/LC_MESSAGES/django.po

```diff
@@ -292,6 +295,9 @@ msgstr ""
 msgid "Courses"
 msgstr ""
 
+msgid "Parcours"
```

Comment:
There is a mistake here, reference string should be the English one
I find it used here :
`Course.add_property('courses', Course.outdoor_courses, _("Parcours"))`
can you change it to "Course" please ?